omkardixit/mentor-website
# Add /mentor registration endpoint

#### Description

Add /profile/mentor registration endpoint. 

#### How to test 

Run locally and hit http://localhost:8080/profile/mentor endpoint with following data : 
```
{
    "user": {
        "name": "Omkar Dixit",
        "email": "dixit.omkar@gamil.com",
        "contactDetails": {
            "phoneNumber": "2024806700",
            "webSite": "www.google.com",
            "linkedIn": "www.linkedin.com",
            "github": "www.github.com"
        }
    },
    "description": "10 years experienced software developer"
}
```

Expected result: Mentor record gets created. 

Try to post document with same email id. 

Expected response : 
Should get 409 conflict.
